_episodes/l1-02-branching.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "Branching"
```

Comment:
This episode should probably have an exercise (address later)

Reply:
New issue: https://github.com/ImperialCollegeLondon/intermediate_grad_school_git_course/issues/63